### ISSUE TITLE
install foundation package failed

### ISSUE THREAD

**Issue by ChinaXing** [typelead/eta-hackage#38]
does foundation package can be supported ? 

https://github.com/haskell-foundation/foundation

```
Configuring foundation-0.0.9...
Preprocessing library for foundation-0.0.9..
Building library for foundation-0.0.9..

Foundation/Primitive/Types.hs:46:18:
    Could not find module ‘GHC.IntWord64’
    Use -v to see a list of the files searched for.
```

**Comment by puffnfresh** [comment]
Looks like it was removed in https://github.com/typelead/eta/commit/6e2488c5deb5f7f45a5fa6652cb209d11193ae8 for some reason.

**Comment by ChinaXing** [comment]
Yes. I wanna use yesod to develop my business system. but found too many package dependencies of yesod install failed when use etlas. so gave up at last.

**Comment by rahulmutt** [comment]
I recommend using Servant which is much more lightweight (or even Spock/Scotty/Happstack) which are pure Haskell and more likely to work with Eta.

**Comment by rahulmutt** [comment]
`foundation` itself will take some time to port and I'll leave this issue open.

**Comment by rahulmutt** [comment]
`foundation`'s patch is available on Eta v0.7. Closing.
